src/App.tsx

```diff
@@ -19,14 +40,37 @@ function App(): React.ReactElement {
       <Container>
         {/* TODO: Consolidate to component for navigation */}
         <StyledNav>
-          <ul>
-            <li>
-              <Link to="/">Home</Link>
-            </li>
-            <li>
-              <Link to="/profile">Profile</Link>
-            </li>
-          </ul>
+          <HomeButton />
+          <NavItem
```

Comment:
Consolidating these into a separate components will help clean up the `App.tsx` from knowing about all these different icons etc but it's also kinda nice to be able to co-locate the path here and the path we give to `Route` components below in a single location 👀 Happy to hear other's thoughts.

Reply:
I am not strongly against your idea but I think it will be better to put the NavItems into a separate NavBar component for readability and maintainability.

Reply:
Let's see how the `App` component turns out in the future so we [avoid hasty abstractions](https://kentcdodds.com/blog/aha-programming), as I think it's better to optimise for code locality than abstraction at this stage that App component itself isn't really doing much. 🙂